obj/tasks.js

```diff
@@ -0,0 +1,11 @@
+module.exports = {
+    //DOM elements
+    createTaskButton : '[class="btn"]',
+    nameFilter : '/html/body/div[1]/div/div[2]/div/div/section/div/div[1]/div/div[1]/div/div/div[1]/div/div/div[1]/div[3]/div/div[2]/div[1]/div/div/div/input',
```

Comment:
OMG, worst xpath ever :D chance of anything changing in this chain is 99% I will help you out with this one

Reply:
try using (//input[@class='field__text field__text--small'])[1] that is a better way to minimize dependency on whole structure. CSS selector is not supporting selecting nth element from array of matching DOM elements